en/flight_controller/pixfalcon.md

```diff
@@ -54,6 +54,18 @@ This board does not have a debug port (i.e it does not have a port for accessing
 Developers will need to solder wires to the board test pads for SWD, and to the STM32F4 (IC) TX and RX to get a console.
 
 
+## Serial Port Mapping
+
+UART | Device | Port
+--- | --- | ---
+UART1 | /dev/ttyS0 | IO debug
+USART2 | /dev/ttyS1 | TELEM1 (flow control)
+USART3 | /dev/ttyS2 | TELEM2 (flow control)
+UART4 | | 
```

Comment:
This one missing too.

Reply:
USART2 | /dev/ttyS1 | TELEM1 (NO flow control)
UART4 is GPS 
There is no U[s]ART3, 5, 6, 7 or 8

Reply:
Thanks. Fixed as per this. Is GPS /dev/ttyS2 ?

Reply:
yes - that should follow the rule, but I do not see a config.

Reply:
OK, so I've assumed it follows the rule. It might be incorrect, but it is more likely to be correct and we can fix if we get a bug report.